packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -16,14 +19,28 @@ const ListEntries = ({
 	className,
 	highlightKey,
 	list,
+	reorderable,
 	column,
 	getItemKey,
 	getItemName,
 	renderItem,
+	moveItem = moveListItem,
```

Comment:
I don't think this property is necessary, `moveListItem` should work for all known cases.

Reply:
I'm thinking this could be useful in cases where the caller would like to be aware of and handle item movements being made, without being forced to deal with just the updated list in `onReorder`. For example if a certain item's position should be immutable. What do you think?

Reply:
I'd prefer to tackle real use cases as they come rather than offering a property that may or may not work.